content/docs/practices/instrumentation.md

```diff
@@ -1,250 +1,250 @@
-the query count, errors and latency of whatever online serving system the cache is in front of.
+the query count, errors and latency of whatever online-serving system the cache is in front of.
 
 #### Collectors
 
-When implementing a non-trivial custom Collector, it's advised to export a
-Gauge for how long the collection took in seconds and another for the number of
+When implementing a non-trivial custom metrics collector, it is advised to export a
+gauge for how long the collection took in seconds and another for the number of
 errors encountered.
 
-This is one of the two cases when it's okay to export a duration as a Gauge
-rather than a Summary, the other being batch job durations. This is as both
+This is one of the two cases when it is okay to export a duration as a gauge
+rather than a summary, the other being batch job durations. This is as both
 represent information about that particular push/scrape, rather than
 tracking multiple durations over time.
 
 ## Things to watch out for
 
-There's some things to be aware of when doing monitoring generally, and also
-with Prometheus-style monitoring in particular.
+There are some general things to be aware of when doing monitoring, and also
+Prometheus-specific ones in particular.
 
 ### Use labels
 
-Very few monitoring systems have the notion of labels and a rules language to
+Few monitoring systems have the notion of labels and an expression language to
 take advantage of them, so it takes a bit of getting used to.
 
 When you have multiple metrics that you want to add/average/sum, they should
 usually be one metric with labels rather than multiple metrics.
 
-For example rather `http_responses_500_total` and `http_resonses_403_total`
-you should have one metric called `http_responses_total` with a `code` label
+For example, rather `http_responses_500_total` and `http_resonses_403_total`,
+create a single metric called `http_responses_total` with a `code` label
 for the HTTP response code. You can then process the entire metric as one in
 rules and graphs.
 
-As a rule of thumb no part of a metric name should ever be procedurally
-generated, you should use labels instead. The one exception is when proxying
+As a rule of thumb, no part of a metric name should ever be procedurally
+generated (use labels instead). The one exception is when proxying metrics
 from another monitoring/instrumentation system.
 
 See also the [naming](../naming) section.
 
-### Don't overuse labels
+### Do not overuse labels
 
-Each labelset is an additional timeseries that has RAM, CPU, disk and network
-costs. Usually this is negligable in the grand scheme of things, however if you
-have lots of metrics with hundreds of labelsets across hundreds of servers this
-can add up quickly.
+Each labelset is an additional timeseries that has RAM, CPU, disk, and network
+costs. In scenarios with lots of metrics and hundreds of labelsets across hundreds
+of servers, this can add up quickly.
```

Comment:
I think we should keep in the fact that this usually doesn't matter, this reduces the changes that the fact that we support labels could be interpreted as a bad thing.